.github/workflows/ci.yml

```diff
@@ -82,6 +82,15 @@ jobs:
     - uses: actions/checkout@v2
     - uses: DeterminateSystems/nix-installer-action@v11
     - uses: DeterminateSystems/magic-nix-cache-action@v6
+    - name: Install Kernel DebugInfo
+      # Install the Debug Info for the current Kernel,
+      # which is required for our runtime tests of kprobe inline functions.
+      run: |
+        # From Ubuntu's Wiki: https://wiki.ubuntu.com/DebuggingProgramCrash#Non-built-in_debug_symbol_packages_.28.2A-dbgsym.29
+        sudo apt-get install ubuntu-dbgsym-keyring
+        echo -e "deb http://ddebs.ubuntu.com $(lsb_release -cs) main restricted universe multiverse\ndeb http://ddebs.ubuntu.com $(lsb_release -cs)-updates main restricted universe multiverse\ndeb http://ddebs.ubuntu.com $(lsb_release -cs)-proposed main restricted universe multiverse" | sudo tee -a /etc/apt/sources.list.d/ddebs.list
```

Comment:
I love the 'restricted universe multiverse' part 😛

Reply:
haha
Ideally, we'd rely on vmtests to test on the Kernel.
Until then, we have to use this solution that can sometimes be flaky 🙁

Reply:
Looks like it takes about a minute and a half to install in CI. And given that CI stability and speed is fairly critical, not sure we should rush this in. Better to direct effort towards vmtest work instead if it's really that desired. 

Kernel dwarf is a tricky to use in open source world anyways - how do users even know to install the debug info anyways? Even if they want to, it's on the order of gigabytes and maximally tricky to install (as evidenced by this block). And that's for distros that even ship debug info.

I'm not objecting to the dwarf support itself - it is a contained change that can be useful if user really needs it. But the CI bit is the uncontained change - it affects the entire project on a daily basis. I'd much rather prefer to get a fast and reliable mechanism in place. 

Perhaps we can run a test like this internally? Should be much less flaky and still provide coverage.